petastorm/tests/test_parquet_reader.py

```diff
@@ -86,3 +85,35 @@ def test_string_partition(reader_factory, tmpdir, partition_by):
         row_ids_batched = [row.id for row in reader]
     actual_row_ids = list(itertools.chain(*row_ids_batched))
     assert len(data) == len(actual_row_ids)
+
+
+@pytest.mark.parametrize('reader_factory', _D)
+def test_column_subset(scalar_dataset, reader_factory):
```

Comment:
Are we covering here functionality not already covered by `test_specify_columns_to_read` test in this file?

Reply:
It was a duplicate test - I've removed it.